### ISSUE TITLE
AIO: display a message when no upcoming events are present

### ISSUE THREAD

**Issue by AndrewKushnir** [angular/angular#40052]
Currently when there are no upcoming events present, an empty table is rendered at https://angular.io/events.

<img width="1600" alt="no-events-page" src="https://user-images.githubusercontent.com/43554145/101679161-c93deb00-3a13-11eb-8a9d-1e1ae5fff43b.png">

The proposal is to render a message with the following content (provided by @mgechev) instead:

```
We don't have any upcoming speaking engagements at the moment.
Until something comes up, make sure you check our <a href="https://www.youtube.com/angular">YouTube channel</a> and follow us on <a href="https://twitter.com/angular">social media</a>.
If you want us to be part of your event reach out on devrel@angular.io!
```

**Comment by petebacondarwin** [comment]
Consider it done :-)

**Comment by AndrewKushnir** [comment]
Thanks @petebacondarwin 👍 

Note: I've just added one more line to the message (also proposed by Minko).

**Comment by petebacondarwin** [comment]
Does this look OK?

<img width="772" alt="Screenshot 2020-12-09 at 20 42 44" src="https://user-images.githubusercontent.com/15655/101684976-1e531e80-3a5f-11eb-9de5-361634c20703.png">

I converted the "Where we'll be..." and "Where we already..." lines to `<h2>` headings (rather than just paragraphs).
And I put the given text into three paragraphs rather than a single one.

**Comment by AndrewKushnir** [comment]
Wow, that was super fast, thanks Pete! This looks good to me 👍

**Comment by mgechev** [comment]
Looks great! Maybe devrel@angular.io could be a mailto link?

**Comment by gkalpak** [comment]
**Fun fact:**
A similar change was implemented in 43e3a8a649803ebc2d3a2777f21f7f2196bbc36d (as part of #36517), but it seems that it was accidentally lost in a subsequent rebase 😱
